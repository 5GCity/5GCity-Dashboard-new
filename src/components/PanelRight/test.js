/**
 * Sidebar Component Tests
 *
 * @author Your Name <youremail@ubiwhere.com>
 */
import React from 'react'
import renderer from 'react-test-renderer'
import Sidebar from './index'
import 'jest-styled-components'

it('renders correctly', () => {
  const fakeData = {
    title:"0020110b0c30c",
    subtitle: "Cabinet"
  }

  const tree = renderer
    .create(<Sidebar data={fakeData} />)
    .toJSON()

  expect(tree).toMatchSnapshot()
})
