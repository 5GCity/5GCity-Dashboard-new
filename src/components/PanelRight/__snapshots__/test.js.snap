// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  position: fixed;
  height: 100%;
  z-index: 1;
  width: 100%;
  display: none;
  background-color: rgba(55,71,79,0.7);
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex: auto;
  -ms-flex: auto;
  flex: auto;
  height: 100%;
  width: 268px;
  background: rgba(55,71,79,0.98);
  position: fixed;
  right: 0px;
}

<div
  className="c0"
  data={
    Object {
      "subtitle": "Cabinet",
      "title": "0020110b0c30c",
    }
  }
>
  <div
    className="c1"
  />
</div>
`;
